CHANGES.rst

```diff
@@ -4,6 +4,14 @@
 Service fixes and enhancements
 ------------------------------
 
+observations
```

Comment:
There should be two changelog entries, one under MAST and one under Gemeni.

Reply:
Would it be a good idea to assign the old name of the class to the new name
 e.g `ObservationsClass = GeminiObservationsClass()`
and leave a deprecation message such as **ObservationsClass is deprecated. Use GeminiObservationsClass instead **

Reply:
> This is a massive breaking change for users, we need to have a deprecation period where the old syntax still works but a deprecation message is displayed. Check out https://docs.astropy.org/en/stable/api/astropy.utils.decorators.deprecated.html#astropy.utils.decorators.deprecated

Would it be a good idea to assign the old name of the class to the new name
`e.g ObservationsClass = GeminiObservationsClass()`
and display a deprecated message saying **ObservationsClass is deprecated. Use GeminiObservationsClass instead**

Reply:
You can use this PR as an example for renaming and deprecating a class: https://github.com/astropy/astropy/pull/9445

Reply:
Nice! Thanks @bsipocz